### PR TITLE
feat: Initialized the repo with a draft streamlit app

### PR DESCRIPTION
**Pull request by frgfm**
This PR introduces the following modifications:
- adds basic streamlit app
- updates README and adds CONTRIBUTING guidelines (along with a Makefile to ease the contribution process)
- added Docker support to run the app
- added CI jobs: ensure that the app builds & runs correctly, that code quality is optimal
- added Github funding button
- added issue templates & environment collection scripts

This is how it renders:
pyro_crowdsourcer

Any feedback is welcome!

### CODE REVIEW

File: .isort.cfg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[settings]
+line_length = 120
+src_paths = doctr,tests
+skip_glob=**/__init__.py
+known_third_party=tensorflow,torch,torchvision,wandb,fastprogress
```
Comment (fe51) [inline .isort.cfg:5]: Hhat is this line for? 
I am not familiar with .isort.cfg

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
+# Contributing to the project
+
+Everything you need to know to contribute efficiently to the project!
+
+Whatever the way you wish to contribute to the project, please respect the [code of conduct](CODE_OF_CONDUCT.md).
```
Comment (fe51) [inline CONTRIBUTING.md:5]: Shall we add this code of conduct in all repos ?

**Review by fe51 — DISMISSED** [review]
Hello there, 

Thanks for initiating the repo ! 

Apart from some questions about the configs (i'm not knowledgeable enough but i know you do :) ), it looks okay to me to start this repo!

Thanks a lot

File: .isort.cfg (hunk 1 above)
Comment (frgfm) [reply .isort.cfg:5]: It's similar to flake8 but specify to import ordering

File: CONTRIBUTING.md (hunk 2 above)
Comment (frgfm) [reply CONTRIBUTING.md:5]: yes

**Pull request merged**